Header header

uint8 COMMAND_WAIT = 0
uint8 COMMAND_SORTBRICKS = 1
uint8 COMMAND_LOADBRICKS = 2
uint8 COMMAND_ABORT = 3
uint8 command

mes_order order
